packages/sil-governance/src/routes/create-proposal/components/FormOptions/ReleaseGuaranteeFunds.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { TokenTicker } from "@iov/bcp";
+import { storiesOf } from "@storybook/react";
+import { useForm } from "medulas-react-components";
+import React from "react";
+
+import DecoratedStorybook, { governanceRoot } from "../../../../utils/storybook";
+import ReleaseGuaranteeFunds from "./ReleaseGuaranteeFunds";
+
+const initialTickers = ["BASH" as TokenTicker, "CASH" as TokenTicker];
+
+storiesOf(governanceRoot, module)
+  .addParameters({ viewport: { defaultViewport: "responsive" } })
+  .add("ReleaseGuaranteeFunds", () => {
+    const { form } = useForm({ onSubmit: () => 0 });
```

Comment:
What does `onSubmit: () => 0` do? I've used `onSubmit: () => {}`. Is there any difference?

Reply:
No, it is exactly the same in this case. Your version returns `void` due to a missing return value. My version returns `0`. However, the return value is ignored anyways, so it does not matter.